### DISCUSSION TITLE
Error reading tiffs - can I override/ignore?

### DISCUSSION THREAD

**Discussion by Jmuccigr** [ImageMagick/ImageMagick#3206] [Help]
I've got some old CDs with images on them and a bunch of them (tiffs) are giving me errors when I try to do something with IM. For example:

```
identify: Unknown field with tag 32768 (0x8000) encountered. `TIFFReadDirectory' @ warning/tiff.c/TIFFWarnings/1046.
identify: file.tif: Bad value 0 for "Orientation" tag. `_TIFFVSetField' @ error/tiff.c/TIFFErrors/661.
```

Apps like GraphicConverter or Preview (Mac guy here) open the images just fine. If I do some minor editing (like rotating the image a net 360°) and re-save it with one of those apps, they seem to correct the error upon saving, so that IM can open the file.

My question then is whether I can force IM to open the file anyway, or if there's a way to get it to report the error more nicely. I have grabbed the filename from the error from a simple bash script, so the latter is more theoretical at this point.

**Comment by snibgo** [comment]
What version of IM, on what platform?

Please post a link to a TIFF that has this problem.

**Reply by Jmuccigr** [reply]
MacOS 10.15.7

Version: ImageMagick 7.0.10-60 Q16 x86_64 2021-01-25 https://imagemagick.org
Copyright: (C) 1999-2021 ImageMagick Studio LLC
License: https://imagemagick.org/script/license.php
Features: Cipher DPC HDRI Modules OpenMP(4.5)
Delegates (built-in): bzlib fftw fontconfig freetype heic jng jp2 jpeg lcms ltdl lzma png tiff webp wmf xml zlib

[Sample file](https://www.dropbox.com/s/6igf4ryjcm25kom/orientation_problem.tif)
